docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -154,27 +159,30 @@ public class ProtectedResource {
 <1> Only requests with OpenID Connect scope `email` are going to be granted access.
 <2> The read access is limited to the client requests with scope `orders_read`.
 
-Please refer to the Permission annotation section of the xref:security-authorize-web-endpoints-reference.adoc#permission-annotation[Authorization of web endpoints]
-guide for more information about the `io.quarkus.security.PermissionsAllowed` annotation.
+For more information about the `io.quarkus.security.PermissionsAllowed` annotation, see the xref:security-authorize-web-endpoints-reference.adoc#permission-annotation[Permission annotation] section of the "Authorization of web endpoints" guide.
 
 [[token-verification-introspection]]
-=== Token Verification And Introspection
+=== Token verification and introspection
 
-If the token is a JWT token then, by default, it will be verified with a `JsonWebKey` (JWK) key from a local `JsonWebKeySet` retrieved from the OpenID Connect Provider's JWK endpoint. The token's key identifier `kid` header value will be used to find the matching JWK key.
-If no matching `JWK` is available locally then `JsonWebKeySet` will be refreshed by fetching the current key set from the JWK endpoint. The `JsonWebKeySet` refresh can be repeated only after the `quarkus.oidc.token.forced-jwk-refresh-interval` (default is 10 minutes) expires.
-If no matching `JWK` is available after the refresh then the JWT token will be sent to the OpenID Connect Provider's token introspection endpoint.
+If the token is a JWT token, then, by default, it is verified with a `JsonWebKey` (JWK) key from a local `JsonWebKeySet`, retrieved from the OIDC provider's JWK endpoint.
+The key identifier (`kid`) header value is used to find the matching JWK key.
+If no matching `JWK` is available locally, then `JsonWebKeySet` is refreshed by fetching the current key set from the JWK endpoint.
+The `JsonWebKeySet` refresh can be repeated only after the `quarkus.oidc.token.forced-jwk-refresh-interval` expires.
+The default expiry time is 10 minutes.
+If no matching `JWK` is available after the refresh, the JWT token is sent to the OIDC provider's token introspection endpoint.
```

Comment:
Same comment as for line 169, for clarity, should we say "...`JWK` key..." here?

Reply:
@sheilamjones - +1. Thanks for flagging here and elsewhere. All instances are now fixed.